main-support.ts

```diff
@@ -184,6 +184,63 @@ const spawn = require('child_process').spawn;
 // also spawns a shell (can pass a single cmd string)
 const exec = require('child_process').exec;
 
+export function checkForCorruptFile(pathToVideo: string,
+  fileHash: string,
+  duration: number,
+  screenshotHeight: number,
+  numberOfScreenshots: number,
+  saveLocation: string,
+  done: any) {
+    if (fs.existsSync(saveLocation + '/filmstrips/' + fileHash + '.jpg')) {
+      // console.log("thumbnails for " + fileHash + " already exist");
+      takeTenClips(pathToVideo,
```

Comment:
Doesn't need to be done now, but what do you think about renaming `takeTenClips` to `extractVideoIfAbsent` (or something similar).

The method name makes it sound like it always generates a video with ten clips (rather than do it conditionally) 😅

Reply:
Sounds good to me! 👍 I'll change it in the next one!